### ISSUE TITLE
You should be able to filter across multiple models with the same field type in the dashboard filters

### ISSUE THREAD

**Issue by TuringLovesDeathMetal** [lightdash/lightdash#1617]
### Description: What is it?

I want to be able to filter multiple dashboard tiles from unrelated explores using one filter. 

The fields would all need to be of the same type. But, I would be able to apply a filter for something like `date`, for example, then select the tiles that I want to apply this filter to.

### Problem: What problem does this solve?

We have many different models with a field for retailer. I’ve created a dashboard of charts we’d like to see for each retailer. Right now I can filter so each chart only shows data for a particular retailer, but I have to either create separate charts for each retailer or add a filter for every model.

### What: Roughly, what does this look like in the product?

In the dashboard filters, when I select a first filter, it should ask you if you want to apply that filter to all tiles with that same field ID on the dashboard. 

It should show you the other tiles/models that will be filtered, and you should be able to select all of the models easily, or select only some of them, or change the "matching" filter in another tile. 

E.g. if I want to filter on my Users explore using the `date` field, then it would ask if I wanted to filter `organizations` on `date` and `app_events` on `date`. I'd be able to select them all, or only select some of them. 

Then, I'd also be able to change the field in a related model to be "matched". E.g. I could click on the `organizations` , `date` and change `date` to be `date_created` or something instead. So now, my `users.date` filter would apply to `organizations.date_created`

**Comment by TuringLovesDeathMetal** [comment]
@PriPatel here's an example:

Our KPI dashboard has a bunch of tiles. I want _**all**_ of the tiles in the dashboard that are time-series charts (i.e. they have a date on the x-axis) to look at the last 14 weeks. 

To do this, I'd start out by picking a filter that's used in a lot of tiles (e.g. `Organizations daily - date week`), then I'd add filters to the rest of the tiles using different fields. 

- Tile A: `date week` vs. `total number of organizations`. Built from `Organizations daily`, so I can just apply the filter I created to this tile. 
- Tile B: `date week` vs. `total number of 7d active organizations`. Built from `Organizations daily`, so I can just apply the filter I created to this tile. 
- Tile C: `Current number of production instances`. Built from `Organizations daily`. I don't want _**any**_ `week` filter applied to this tile. 
- Tile D: `date week` vs. `total number of 7d active users`. Built from `Users daily`. So, I want to apply the `week` filter to this, but I want to change the field it's filtering on to `Users daily - date week`
- Tile E: `Total number of community members`. Built from `community stats daily. I don't want _**any**_ `week` filter applied to this tile.
